### ISSUE TITLE
Avoid unnecessary use of heading elements

### ISSUE THREAD

**Issue by blnkt** [lsst-epo/rubin-obs-client#68]
## Description
Heading elements should be utilized only when necessary and primarily to mark the beginning of related sections of content. Making an effective use of headings can provide an excellent benefit to users of assistive technology, which may provide the user with a way to navigate section-by-section through content. By applying headings incorrectly, or for passages of text which do not delineate a section, users of assistive technology may be stripped of this useful navigation technique.

### Event Page
- [ ] Event Page Pretitle should be styled as it is, but should be a `div` instead of an `h4`.

![21017dc62613ef4e209076272b9e37a332dc01e6-1641292344](https://user-images.githubusercontent.com/8166060/153070178-7a224421-3132-4717-b6fc-b40c1ee9c8b0.png)

### Staff Profile Page
- [x] Staff Profile Pretitle should be styled as it is, but should be a `div` instead of an `h4`

<img width="769" alt="cd129dea3eaf0c94f4d12b045595380af828611c-1641219067" src="https://user-images.githubusercontent.com/8166060/153070323-eff1128d-d97e-403b-b042-941e9f47ebf6.png">

## CIC Estimate
0 points

**Comment by dananjohnson** [comment]
Resolved by 74c9f29
